go/bundle/decoder.go

```diff
@@ -275,6 +276,91 @@ func parseIndexSection(sectionContents []byte, sectionsStart uint64, sos []secti
 	return requests, nil
 }
 
+// https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html#index-section
+func parseIndexSection(sectionContents []byte, sectionsStart uint64, sos []sectionOffset) ([]requestEntryWithOffset, error) {
+	// Step 1. "Let index be the result of parsing sectionContents as a CBOR item matching the index rule in the above CDDL (Section 3.5). If index is an error, return an error." [spec text]
+	dec := cbor.NewDecoder(bytes.NewBuffer(sectionContents))
+	numUrls, err := dec.DecodeMapHeader()
+	if err != nil {
+		return nil, fmt.Errorf("Failed to decode index section map header: %v", err)
```

Comment:
Optional Nit: make err msg prefix consistent with others?
"bundle.index: Failed to..."

Reply:
Done.